assets/Bootstrap/gulpfile.js

```diff
@@ -0,0 +1,92 @@
+var gulp = require('gulp');
+var less = require('gulp-less');
+var browserSync = require('browser-sync').create();
+var header = require('gulp-header');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2M691381/blog/pullRequest?prid=8438001)
```suggestion
var header = require("gulp-header");
```